src/js/components/Form/index.d.ts

```diff
@@ -1,16 +1,17 @@
 import * as React from "react";
 
-export interface FormProps {
+export interface FormProps<T> {
   errors?: {};
   infos?: {};
   messages?: {invalid?: string,required?: string};
-  onChange?: ((event: React.ChangeEvent) => void);
+  onChange?: (value: T) => void;
```

Comment:
@ericsoderberghp , I think this PR is good to go, just wanted you to give your inputs on this change and let us know if there are any concerns.

Reply:
no concerns from me